actors/miner/src/lib.rs

```diff
@@ -1216,12 +1214,12 @@ impl Actor {
                     new_sector_info.deal_ids = with_details.update.deals.clone();
                     new_sector_info.activation = rt.curr_epoch();
 
-                    new_sector_info.deal_weight = with_details.deal_weights.deal_weight.clone();
-                    new_sector_info.verified_deal_weight = with_details.deal_weights.verified_deal_weight.clone();
-
-                    // compute initial pledge
                     let duration = with_details.sector_info.expiration - rt.curr_epoch();
```

Comment:
This duration _must_ be the sector info's `expiration - activation`, or we won't be able to get the right size back. I think it is, but it's not obvious and someone could easily mess this up. Please change it to `new_sector-info.expiration - new_sector_info.activation` and comment as such.